newspaper/network.py

```diff
@@ -71,7 +71,10 @@ def get_html_2XX_only(url, config=None, response=None):
     return html
 
 
-def _get_html_from_response(response):
+def _get_html_from_response(response, config=None):
+    for disabled_type in config.ignored_content_types_defaults:
+        if disabled_type in response.headers.get('content-type'):
```

Comment:
As per the requests library API, `response.headers.get('content-type')` returns a string that represents the content-type. 

```
>>> resp = requests.get('https://www.adobe.com/pdf/pdfs/ISO32000-1PublicPatentLicense.pdf')
>>> resp.headers.get('content-type')
'application/pdf'
```

Why perform a substring match with `in` instead of just string comparison via `==`? Also this is not ideal for runtime complexity since you are checking every ignored content type in a loop, instead you can perform this instant lookup:

```
if response.headers.get('content-type') in config.ignored_content_types_defaults:
  return ''
```

Reply:
Fixed